### DISCUSSION TITLE
Hi I just installed NocoDB using npm and [Sep 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3902] [Discord Threads]
> 🙋‍♂️ **Mud_2019**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Mud_2019**  
#### Do I just modify the NC_DB environment variable to reflect the IP of my machine where NocoDB is hosted?  
#  
> 🙋‍♂️ **Mud_2019**  
#### NC_BACKEND_URL    No    Custom Backend URL    http://localhost:8080 will be used  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1020786502323666984)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
